src/app/components/action-bar/wizard-bar.jsx

```diff
@@ -80,54 +80,55 @@ function ChplWizardBar(props) {
   return (
     <ThemeProvider theme={theme}>
       <div className="action-bar">
-        { isConfirming
+        {isConfirming
           && (
-          <ChplActionBarConfirmation
-            dispatch={handleConfirmation}
-            pendingMessage={pendingMessage}
-          />
+            <ChplActionBarConfirmation
+              dispatch={handleConfirmation}
+              pendingMessage={pendingMessage}
+            />
           )}
-        { (errors?.length > 0 || warnings?.length > 0)
+        {(errors?.length > 0 || warnings?.length > 0)
           && (
             <>
               <div className="action-bar__error-toggle">
-                <span
+                <div
                   onClick={() => setShowMessages(!showMessages)}
                   onKeyDown={() => setShowMessages(!showMessages)}
                   tabIndex={0}
                   role="button"
                 >
-                  { errors?.length > 0
+                  {errors?.length > 0
                     && (
                       <>
                         Error
-                        { errors.length > 1 && 's'}
+                        {errors.length > 1 && 's'}
                       </>
                     )}
-                  { errors?.length > 0 && warnings?.length > 0
+                  {errors?.length > 0 && warnings?.length > 0
                     && <> and </>}
-                  { warnings?.length > 0
+                  {warnings?.length > 0
                     && (
                       <>
                         Warning
-                        { warnings.length > 1 && 's'}
+                        {warnings.length > 1 && 's'}
                       </>
                     )}
-                  <i className={`fa ${showMessages ? 'fa-caret-down' : 'fa-caret-left'}`} />
-                </span>
+                  <span className={classes.iconSpacing}>
+                    <i className={`fa ${showMessages ? 'fa-caret-down' : 'fa-caret-left'}`} /></span>
```

Comment:
Here's an example of something that should be converted; this is using a FontAwesome icon, but we should be converting to the MUI Icon sets wherever we can, especially if we're creating a new component entirely